src/globals.ts

```diff
@@ -1,12 +1,123 @@
 import * as logger from './logger';
 import { Context } from 'aws-lambda';
-
 const MAX_TAGS = 50;
 const MAX_TAG_KEY_LEN = 50;
 const MAX_TAG_VALUE_LEN = 50;
 const ADD_TAG_ERROR_MSG_PREFIX = 'Skipping addExecutionTag: Unable to add tag';
 export const DEFAULT_MAX_SIZE_FOR_REQUEST = 1000 * 1000;
 
+export const Timer = (() => {
```

Comment:
Try to find a better name, its not just a timer that anyone can use
```suggestion
export const RuntimeTimer | 
export const GlobalTimer |
export const ... |
```